tests/unittests/tensorsTest.cpp

```diff
@@ -485,55 +485,6 @@ TEST(Tensor, externallyManagedPayload) {
   EXPECT_EQ(int(payload[3]), 30);
 }
 
-/// Broadcast a Tensor of shape (2,1) to (3,2,4,2) with axis 1.
-TEST(Tensor, broadcastNewShape) {
```

Comment:
Are there any existing tests covering correctness of new broadcast impl?

Reply:
Yes, there are tests in OperatorTest that create BroadcastNodes.